viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -203,6 +204,13 @@ export class Cognite3DViewer {
   }
 
   constructor(options: Cognite3DViewerOptions) {
+    const threejsRequiredVersion = viewerPackageJson.peerDependencies.three.split('.')[1].toString();
+    if (threejsRequiredVersion != THREE.REVISION) {
+      log.warn(
+        'Loaded ThreeJS version is different from Reveal depended version. Upgrade ThreeJS version to ',
+        threejsRequiredVersion
+      );
```

Comment:
Maybe something like this is better (note that I haven't tested this)? This would also include the patch version which could be relevant. Note that linting is probably messed up here.

```suggestion
      log.warn(
        `The version of the dependency \"three\" is different from what Reveal expects, which may cause unexpected results. In case of unexpected issues, please set the version to ${viewerPackageJson.peerDependencies.three}`  
      );
```